docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -4,14 +4,55 @@
+    "*Estimated QPU usage: 14 seconds (tested on IBM Sherbrooke)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background: What is a pulse gate?\n",
+    "\n",
+    "A quantum gate is an abstraction for the operation applied to qubits. In superconducting qubit architecture, its physical implementation typically involves applying microwave pulses - these are electromagnetic waves at specific frequencies with certain amplitudes and durations - to interact with qubits and control the qubit states. Those microwave pulses are pre-calibrated to implement a desired operation and serve as one of the [standard native gates](https://docs.quantum.ibm.com/run/native-gates). The Qiskit OpenPulse feature allows users to access pulse-level control and use _pulse gates_ - gates that are specified by pulse schedules.\n",
+    "\n",
+    "Pulse gates provide more flexibility in defining and fine-tuning qubit operations and could potentially be used to construct gates that have higher fidelities, particularly when translating circuits for quantum simulation with small Trotterization time steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This tutorial illustrates how to transpile circuits in a pulse-efficient way by transpiling two-qubit gates into the [`RZXGate`](https://qiskit.org/documentation/stubs/qiskit.circuit.library.RZXGate.html).  The process consists of applying echoed cross-resonance pulses with varying duration to turn on the two-qubit $Z\\otimes X$ interaction. This implements the 'RZXGate' with an arbitrary rotation angle $\\theta$.  Because it is implemented by directly specifying the gate's pulse-level instruction schedule and requires no further decomposition, it serves as a continuous gate set native to the hardware.\n",
+    "\n",
+    "The following examples show that this method results in shorter circuit depth and duration. Because qubits have a limited coherence time, a shorter total circuit duration is desirable. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The pulse-efficient transpilation has been shown to give more accurate results in the quantum approximate optimization algorithm applied to MAXCUT (see reference [here](https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.3.043088)). This notebook demonstrate how to implement the described technique to transpile a relevant circuit in simulating quantum dynamics. The result from the hardware suggests that the `RZXGate` type of transpilation produces more accurate results."
```

Comment:
```suggestion
    "The pulse-efficient transpilation has been shown to give more accurate results in the quantum approximate optimization algorithm applied to MAXCUT (see reference [here](https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.3.043088)). This tutorial demonstrates how to implement the described technique to transpile a relevant circuit in simulating quantum dynamics. The result from the hardware suggests that the `RZXGate` type of transpilation produces more accurate results."
```
In general could we use the term "tutorial" for consistency. For people just reading this tutorial in the website it might not be clear that this is a jupyter notebook.

Reply:
Also in this section, is there any way we can mention more explicitly how this is useful for utility scale work?